docs/docs/deploy/baremetal.md

```diff
@@ -24,9 +30,12 @@ The baremetal deploy runs several commands in sequence. These can be customized,
 3. `yarn rw prisma generate` - generates latest Prisma Client libs
 4. `yarn rw dataMigrate up` - runs data migrations, ignoring them if not installed
 5. `yarn rw build` - builds the web and/or api sides
-6. `yarn pm2 restart [service]` - restarts the serving process(es)
+6. `yarn pm2 restart [service]` - restarts the serving process(es).
```

Comment:
When we've got a bullet list like this we either put a period on all of them or none of them